atlas_splitter/barrel_splitter/somatosensory_barrels.py

```diff
@@ -41,10 +39,10 @@ def layer_ids(
     new_ids: Dict[str, Dict[str, int]] = {}
     for name in names:
         new_ids[name] = {}
-        new_ids[name][name] = id_from_acronym(region_map, name)
```

Comment:
Why are you returning a negative id?

Reply:
It's just the best `int` placeholder I could think of.